src/iframe-helper.js

```diff
@@ -214,7 +214,9 @@ function registerGlobalListenerIfNeeded(parentWin) {
       return;
     }
 
-    let listeners = listenForEvents[data['type']];
+    // SafeFrame's postmessaging protocol specifies the message type under
+    // parameter 's'.
+    let listeners = listenForEvents[data['type'] || data['s']];
```

Comment:
Do we expect to handle all possible message formats here?  Seems brittle... ideally we could specify the format with the listener function

Reply:
Done.